src/patternfly/components/Table/table.scss

```diff
@@ -212,20 +275,39 @@
 
   // Table header cell
   thead {
-    --pf-c-table-cell--PaddingTop: var(--pf-c-table-thead-cell--PaddingTop);
-    --pf-c-table-cell--PaddingBottom: var(--pf-c-table-thead-cell--PaddingBottom);
-    --pf-c-table-cell--FontSize: var(--pf-c-table-thead--FontSize);
-    --pf-c-table-cell--FontWeight: var(--pf-c-table-thead--FontWeight);
-
-    white-space: nowrap;
-    vertical-align: top;
+    --pf-c-table--cell--FontSize: var(--pf-c-table--thead--cell--FontSize);
+    --pf-c-table--cell--FontWeight: var(--pf-c-table--thead--cell--FontWeight);
+    --pf-c-table--cell--Width: var(--pf-c-table--thead--cell--Width);
+
+    // Set default th behavior to truncate
+    --pf-c-table--cell--MinWidth: var(--pf-c-table--m-truncate--cell--MinWidth);
+    --pf-c-table--cell--MaxWidth: var(--pf-c-table--m-truncate--cell--MaxWidth);
+    --pf-c-table--cell--Overflow: var(--pf-c-table--m-truncate--cell--Overflow);
+    --pf-c-table--cell--TextOverflow: var(--pf-c-table--m-truncate--cell--TextOverflow);
+    --pf-c-table--cell--WhiteSpace: var(--pf-c-table--m-truncate--cell--WhiteSpace);
+
+    // Align content to bottom
+    --pf-c-table__button-content--AlignItems: var(--pf-c-table--thead__button-content--AlignItems);
```

Comment:
does it help for this to be a var? Looks like it's only used once in a property declaration, and the variable here as the value is only declared once.